README.md

```
@@ -368,6 +372,34 @@ When crawling in headless mode, additional chrome options can be specified using
 katana -u https://tesla.com -headless -system-chrome -headless-options --disable-gpu,proxy-server=http://127.0.0.1:8080
 ```
 
+*`-page-load-strategy`*
```

markdownlint-cli2 (0.21.0)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)